cdc/kv/client.go

```diff
@@ -271,6 +271,28 @@ func (rl *regionEventFeedLimiters) getLimiter(regionID uint64) *rate.Limiter {
 	return limiter
 }
 
+// CDCKVClient is an interface to receives kv changed logs from TiKV
+type CDCKVClient interface {
+	EventFeed(
+		ctx context.Context,
+		span regionspan.ComparableSpan,
+		ts uint64,
+		enableOldValue bool,
+		lockResolver txnutil.LockResolver,
+		isPullerInit PullerInitialization,
+		eventCh chan<- *model.RegionFeedEvent,
```

Comment:
Would it be better if the writer "owned" the channel?

Reply:
Do you mean the kv client creates a channel and passes it back to the outer caller? I prefer to keep current way, maybe in the future the eventCh could be shared by multiple KV writer

Reply:
Okay, seems fair.